src/lib/rmt/base.rb

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2018 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require 'yaml'
+
+module RMT
+end
+
+class RMT::Base < Yast::Client
+  include Yast::UIShortcuts
+  include Yast::Logger
+  include Yast::I18n
+
+  CONFIG_FILENAME = '/etc/rmt.conf'.freeze
```

Comment:
In my case this file did not exist (I only have the development setup) and breaks only after going through the entire process. Do we want to catch the non-existence or is it always silently assumed `/etc/rmt.conf` is there?

Reply:
We might want to check if it exists, because the config that is shipped with the package contains some sane default values.
Otherwise we'd need to create a default config and populate it with some values.

Reply:
I tested the changes by pointing to another, writable location for the config file and it worked as expected. In the expected setup YaST should have access to `/etc` so I think it's fine (question to YaST team maybe? :/)